README.md

```diff
@@ -2164,27 +2165,29 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
 that depends greatly on your skills, the marketplace of candidates, what other offers you
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
-However, this section covers some basics of what to expect with offers, and tips on
+However, this section covers some basics of what to expect with offers, as well as tips on
 [negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
 
 ### Equal treatment
 
 🔹 Companies can and should work hard to ensure that all candidates are
 [given equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/)
-in the hiring process but
+in the hiring process, but
 [inequalities persist](https://iwpr.org/publications/gender-wage-gap-2017-race-ethnicity/).
 [Workplace disparities](https://digitalcommons.ilr.cornell.edu/cgi/viewcontent.cgi?article=2208&context=articles)
 in pay and opportunity span race and gender, with research focusing on
 [inequality in the U.S. workplace](http://www.pewresearch.org/fact-tank/2016/07/01/racial-gender-wage-gaps-persist-in-u-s-despite-some-progress/),
 [executive leadership](https://pr-paywatch-aflcio.pantheonsite.io/paywatch/company-pay-ratios)
-and its well documented
+and its well-documented
 [lack of diversity](http://fortune.com/2017/06/09/white-men-senior-executives-fortune-500-companies-diversity-data/),
 and the [technology industry](https://www.eeoc.gov/eeoc/statistics/reports/hightech/).
 [Gender bias](https://www.newyorker.com/science/maria-konnikova/lean-out-the-dangers-for-women-who-negotiate)
-in negotiation is also an issue, making many women feel uncomfortable negotiating.
+in negotiation itself is also an issue;
```

Comment:
Looks good, and look fine to me, so def let's keep. But a process note, these couple changes are edging beyond the assigned "copy-edits-only" tag into content changes, i.e. things that would benefit from content or subject matter (not copy edit) review.

Reply:
Yes, a content change! Felt it was an urgent correction, but totally get it. Thanks for stetting.

Reply:
Yeah. @Hopeopolis do you have any notes on the changes on (only) just these couple changes in this subsection? We merged this along with the copy edits but would be good to get your review as well as this was your material.